js/reducers/user.js

```diff
@@ -85,7 +93,9 @@ export default function user(
 
   if (action.type === USER_LOGIN_ERROR_ACTION && state.isLoggedIn === false) {
     return {
-      loginError: true
+      isLoggedIn: false,
```

Comment:
@fpersico I wonder why in the previous PR the build didn't fail, do we have a step that checks the code with Flow?

Reply:
actually it did fail, I think we need a more verbose output in the build to see what failed (`--info` should be sufficient)